public/src/components/channelManagement/tickerEditor.tsx

```diff
@@ -149,13 +149,25 @@ const TickerEditor: React.FC<TickerEditorProps> = ({
                 <FormControlLabel
                   value={TickerName.US_2022}
                   control={<Radio />}
-                  label="US_2022"
+                  label="US_2022 (do not use)"
```

Comment:
do we need to include the old ones here?

Reply:
I’m not certain, but I have stopped looking at this for now. Will investigate when I revisit.

Reply:
The tickers are not currently in use, so I’ve removed the old names now